meson.build

```diff
@@ -82,6 +82,12 @@ if with_docs
   endif
 endif
 
+with_manpage = get_option('with_manpage')
+help2man = find_program('help2man', required: with_manpage)
+if not help2man.found()
+  help2man = disabler()
+endif
```

Comment:
These three lines are not necessary if you make `with_manpage` a `feature` option instead of a boolean.